test/test-widgets.js

```diff
@@ -155,8 +155,8 @@ test('multipleCheckbox', function (t) {
         var field = {
             choices: {
                 one: 'Item one',
-                three: 'Item three',
-                two: 'Item two'
+                two: 'Item two',
```

Comment:
hm, perhaps i disabled the setting to enforce alphabetized keys - if the travis tests pass then it's good as-is